src/dropdown/dropdown.scss

```diff
@@ -217,7 +220,10 @@ $height-small: 30px !default;
   &.is-open {
     background: $background-open;
     border-color: $border-color-open;
-    color: $text-color-open;
+
+    &:hover > .Select-control .Select-value .Select-value-label {
+      color: $text-color-open;
```

Comment:
whoops... actually this IS wrong, and that's why I looked for another solution better than using `&.has-value`